package.json

```diff
@@ -10,13 +10,5 @@
   "precommit": "lerna run --concurrency 1 --stream precommit",
   "scripts": {
     "deploy": "lerna bootstrap --npm-client=yarn && lerna publish",
-    "prettier": "prettier --write 'packages/**/*.{js,jsx,json,css,md}'",
-    "css": "yarn --cwd packages/css start",
-    "css:setup": "yarn --cwd packages/css install",
-    "react": "yarn --cwd packages/react start",
```

Comment:
Don't we still need those scripts?

Reply:
They're part of the second PR with the rest of the changes to run the code